config/locales/en.yml

```diff
@@ -596,6 +596,7 @@ en:
   alphabetical_order: Alphabetical Order
   already_have_an_account_log_in_html: Already have an account? <strong>Log in.</strong>
   amphibians: amphibians
+  analyze_clashes: Analyze Clashes
```

Comment:
This and every other sentence fragment needs a comment explaining what it is. What's a clash? What do you mean by analyze?

Reply:
added comments

Reply:
This is still pretty confusing to me. If you can't explain it succinctly, I think you need to link to your tutorial. I guarantee you this is going confuse translators, which means more time spent answering questions. I also suggest using a more specific term for a "clash" like "taxon change clash" or "identification clash" and then use that term consistently.